core-bundle/contao/themes/flexible/styles/basic.css

```diff
@@ -1056,13 +1029,34 @@ a.tl_submit {
 	.split-button > button[type="button"]:focus {
 		outline: none;
 	}
+
+	.widget-group > .w25,
+	.widget-group > .w33,
+	.widget-group > .w50,
+	.widget-group > .w66,
+	.widget-group > .w75 {
+		grid-column: auto / span 6;
+	}
 }
 
 /* Handheld */
-@media (max-width: 767px) {
-	.w25, .w33, .w50, .w66, .w75 {
-		float: none;
-		width: calc(100% - 30px);
+@media (max-width: 767.98px) {
+
+	.widget-group {
+		grid-column-gap: 0;
+	}
+
```

Comment:
```suggestion
```

This is not necessary if there are no actual columns (all have span 12).

Reply:
This one is necessary due to the column-gap being 30px by default and enlarging the row on mobile devices (12x30px + margin + paddings).

We could minmax this but it may lead to other layouts than they have been before.

Reply:
But on mobile there are no columns, everything is `span 12` - so the `column-gap` isn't applied anywhere.

Reply:
After your latest changes :)

Reply:
![image](https://github.com/contao/contao/assets/55794780/a50d28c5-a0a2-4792-8e9b-882235556f3e)

That's what happens till 410px (due to the margin (10px * 2) + padding (15px * 2) + gap (12 * 30px) ), thus leaving it will fix the issue. this CSS is already desktop first so resetting it won't hurt and saves one additional media-query.

Reply:
I am still not sure what you mean. There is no visible difference - as it makes no difference, since all widgets span the full 12 columns. Resetting the `column-gap` to `0` can be removed.

Reply:
I think he means that the grid is too wide then (as you can see in the screenshot) as the gaps are always there even if all widgets span across them.

Reply:
Ah sorry, I see now. After a certain width the column size is `0` and the `column-gap`s create a minimum width.